modules/phase_field/tests/KKS_system/kks_multiphase.i

```diff
@@ -291,6 +298,13 @@
     args      = 'eta1 eta3'
     mob_name  = 1
   [../]
+  [./mult_CoupledACint_2]
+    type = SimpleCoupledACInterface
```

Comment:
Well, a lot of what's in this test is specific to the KKS multi-phase system. So I think it makes more sense for the kks_multiphase test to stay with the other KKS_system tests. However I could also add a separate, simpler test that's more specific to SimpleCoupledACInterface and put it with SimpleACInterface, what do you think?